client/src/environments/environment.prod.ts

```diff
@@ -7,5 +7,7 @@ export const environment = {
   DOMAIN_WHITELIST: 'api.ingest.archive.data.humancellatlas.org,ingest.data.humancellatlas.org',
   // AAI
   AAI_CLIENT_ID: '',
-  AAI_AUTHORITY: 'https://login.elixir-czech.org/oidc'
+  AAI_AUTHORITY: 'https://login.elixir-czech.org/oidc',
+
+  OLS_URL: 'https://ontology.staging.archive.data.humancellatlas.org'
```

Comment:
Does this mean we are using OLS Staging for prod?
Or will this be set by configuration later?

Reply:
No, should use prod for prod. I'm not sure for other dev envs.

Reply:
Actually, the configuration is also needed in the ingest-kube-deployment UI template. This environment.prod.ts is only being used locally. The environment.env.ts is the one being used in the deployment.

Reply:
i think it's better to use the corresponding ontology env. Fixed.